tests/platforms/kubernetes/appmanager.go

```diff
@@ -120,7 +120,7 @@ func (m *AppManager) ValidiateSideCar(app utils.AppDescription) (bool, error) {
 			}
 		}
 		if !daprdFound {
-			return false, fmt.Errorf("Cannot find dapr sidecar in %s pod", pod.Name)
+			return false, fmt.Errorf("cannot find dapr sidecar in %s pod", pod.Name)
```

Comment:
Rephrase to `cannot find dapr sidecar in pod %s`

Reply:
Fixed.